content/blog/2020-03-01-bookdown-learnings/index.Rmd

```diff
@@ -0,0 +1,266 @@
+I followed usethis [setup instructions](https://usethis.r-lib.org/articles/articles/usethis-setup.html) to edit my `.Rprofile` so that its functions would be available without explicitly calling the package.
+I also followed the rest of the setup instructions somewhat religiously (e.g. "Prepare your system to build packages from source"), because I wasn't confident enough to make a judgement on what I would and would not need.
+
+Practically, I kept the usethis [pull request helpers](https://usethis.r-lib.org/articles/articles/pr-functions.html) article open in my browser and forced myself to use them.
+At the start of the day, I would open our bookdown RStudio project by clicking my local `ropensci-blog-guidance.Rproj` file, pull to update my local master (because Maëlle would have reviewed, edited and merged my pull requests while I slept), create a new branch with `pr_init(branch = "branchname")`, make edits, and `pr_push()` to push my local changes to GitHub master.
+usethis automatically opened a browser to the GitHub web interface prompting me to open a pull request. 
+
+There's no magic conclusion here. 
+This feels a bit better than working on pull requests on the command line. 
+I expect by forcing myself to usethis I'll discover some magic soon enough.
+
+#### 4. How to fill up a bunch of Rmd files to make a book
+
+To answer the question, "how do I do this thing?" I often compared the 
+GitHub file structure and contents of completed books with their corresponding live pages. This, for me, had the biggest payoff in learning how to bookdown.
+
+Things I learned:
+
+- One book chapter is made from one Rmd file.
+
+- Chapters are woven together in the [`_bookdown.yml`](https://github.com/ropensci-org/blog-guidance/blob/master/_bookdown.yml) file that references those Rmd files
+
+- Create links inside the book with `{#anchor}`. In the [`authorcontent.Rmd`](https://github.com/ropensci-org/blog-guidance/blob/master/authorcontent.Rmd) file, the heading `# Content Guidelines {#content}` means that a markdown format `[link to that chapter](#content)` anywhere in the book will link to the Content Guidelines chapter.
+
+- Appendices A to H (_too many; we know_) are created from a single [appendix.Rmd](https://github.com/ropensci-org/blog-guidance/blob/master/appendix.Rmd) file made up of groupings of a heading, some text, and sometimes a code chunk that points to a file, like a [template](https://github.com/ropensci-org/blog-guidance/tree/master/templates) or [checklist](https://github.com/ropensci-org/blog-guidance/tree/master/checklists) that populates that appendix.
+
+<!--html_preserve--> {{< figure src = "file-comp.png" width = "800" alt = "this is the alternative text" >}}<!--/html_preserve-->
```

Comment:
Image is blurry. Can you suggest a better way? Having one on top of other, instead of side by side prob takes up too much space.

Need to add alt text once decided

Reply:
would an image like that be good for twitterImg? would have to have right proportions

Reply:
I don't find it blurry? I think it's fine?